tests/test_loadDiaCatalogs.py

```diff
@@ -264,7 +264,7 @@ def setUp(self):
                                   self.dateTime)
 
     def tearDown(self):
-        del self.tmp_file
+        os.close(self.db_file_fd)
         os.remove(self.db_file)
         del self.db_file
```

Comment:
I don't think this del does anything useful.

Reply:
I'll remove it